test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -505,8 +505,8 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
     }
 
     $testCase = @(
-        @{ proxy_address = (Get-WebListenerUrl).Authority; name = 'http_proxy'; protocol = 'http' }
-        @{ proxy_address = (Get-WebListenerUrl -https).Authority; name = 'https_proxy'; protocol = 'https' }
+        @{ proxy_address = (Get-WebListenerUrl).Authority; name = 'http proxy'; protocol = 'http' }
```

Comment:
If we are going to use soft names, we should use `HTTP proxy` and `HTTPS proxy`.

Reply:
Thanks! Fixed.